e2e-tests/package.json

```diff
@@ -28,6 +30,13 @@
     "typescript": "5.5.4"
   },
   "dependencies": {
-    "node-fetch": "3.3.2"
+    "@azure/identity": "^4.4.1",
+    "@keycloak/keycloak-admin-client": "^25.0.2",
+    "@kubernetes/client-node": "^0.21.0",
+    "@microsoft/microsoft-graph-client": "^3.0.7",
+    "isomorphic-fetch": "^3.0.0",
+    "node-fetch": "3.3.2",
+    "octokit": "^4.0.2",
```

Comment:
can we replace `isomorphic-fetch`, `node-fetch` and `octokit` and use the built-in functions from @playwright/test for API calls instead of a third-party library. One key reason is that it provides better assertions/logging in the HTML reporter and Trace View, and makes debugging easier.

Reply:
`isomorphic-fetch` and `node-fetch` are both required by the msgraph and keycloak-admin client which is essential to manage the azure and rhsso users and groups during the execution. 
`octokit` is the only official client I found for github teams/users/orgs management, which is also essential for these scenarios. 
Re-writing them as standard rest call would be a lot of work and will need to maintained, but it's critical it can be done. Maybe we can just improve the logs: all the steps are logging to an external fine since the instances are destroyed and recreated multiple times.